src/app/MessageDef/StatusIB.h

```diff
@@ -117,5 +117,15 @@ struct StatusIB
 
 }; // struct StatusIB
 
+constexpr bool operator==(const StatusIB & one, const StatusIB & two)
+{
+    return one.mStatus == two.mStatus && one.mClusterStatus == two.mClusterStatus;
```

Comment:
mClusterStatus is meaningless unless mStatus has one of the two "generic" values, right?

Reply:
If true that's not enforced or validated by the constructor in any way, you can set an arbitrary mStatus with an arbitrary mClusterStatus.

Reply:
Yes, that is true.  This is probably fine, but I can think of weird situations where this can lead to "wrong" results....